### PR TITLE
chore: update edx-event-bus-kafka

### PR DESCRIPTION
**Pull request by rgraber**
Upgrade the edx-event-bus-kafka library and import the correct classes. Also, only check that confluent-kafka is correctly installed since edx-event-bus-kafka is now in the normal requirements.

### CODE REVIEW

**Review by dianakhuang — APPROVED** [review]

File: course_discovery/apps/course_metadata/management/commands/consume_events.py [hunk 1]
```diff
@@ -48,15 +51,9 @@ def add_arguments(self, parser):
...
     def handle(self, *args, **options):  # pragma: no cover
         try:
-            # FIXME: This needs to be replaced with a configurable loading
-            # mechanism built into openedx-events.
-            # See https://github.com/openedx/openedx-events/pull/85)
-            from edx_event_bus_kafka.consumer.event_consumer import \
-                KafkaEventConsumer  # pylint: disable=import-outside-toplevel
-            from edx_toggles.toggles import SettingToggle  # pylint: disable=import-outside-toplevel
-            from openedx_events.tooling import OpenEdxPublicSignal  # pylint: disable=import-outside-toplevel
+            import confluent_kafka  # pylint: disable=import-outside-toplevel,unused-import
```
Comment (Ali-D-Akbar) [inline course_discovery/apps/course_metadata/management/commands/consume_events.py:54]: where does this `confluent_kafka` come from? Is this installed somewhere along with the edx event bus kafta?

**Review by Ali-D-Akbar — APPROVED** [review]
Just a question otherwise looks good.

File: course_discovery/apps/course_metadata/management/commands/consume_events.py (hunk 1 above)
Comment (rgraber) [reply course_discovery/apps/course_metadata/management/commands/consume_events.py:54]: This is installed in the private dockerfile. See https://github.com/edx/internal-dockerfiles/pull/64 for the whole story

Comment (Ali-D-Akbar) [reply course_discovery/apps/course_metadata/management/commands/consume_events.py:54]: Ohh, right so basically it's already installed in the image already. Got it.

**Review by DawoudSheraz — APPROVED** [review]

**Pull request merged**